ice40/make/tests.mk

```diff
@@ -61,16 +61,19 @@ endif
 # ---------------------------------------------
 # ---------------------------------------------
 
-INPUT_IO_FILE=$(TEST_DIR)/$(BOARD).pcf
+INPUT_IO_FILE=$(wildcard $(TEST_DIR)/$(BOARD).pcf)
```

Comment:
It's not clear when we'd need wildcard. Is there a case when a * would be in the BOARD?

Reply:
`$(wildcard xxx)` will return empty when the file doesn't exist.

Reply:
I think this is clearer
`INPUT_IO_FILE=$(if $BOARD,$(TEST_DIR)/$(BOARD).pcf)`